src/Schema.php

```diff
@@ -254,6 +254,11 @@ public static function cron_ensure_indexes() {
     if ($wpdb->get_var($wpdb->prepare('SHOW TABLES LIKE %s', $wpdb->prefix . 'pmxi_hash'))) {
       self::ensureIndex('pmxi_hash', 'post_id', 'post_id');
     }
+
+    // real-media-library lacks an index on its lookup table.
+    if ($wpdb->get_var($wpdb->prepare('SHOW TABLES LIKE %s', $wpdb->prefix . 'wp_realmedialibrary_posts'))) {
+      self::ensureIndex('wp_realmedialibrary_posts', 'fid', 'fid');
```

Comment:
Duplicate table prefix
```suggestion
    if ($wpdb->get_var($wpdb->prepare('SHOW TABLES LIKE %s', $wpdb->prefix . 'realmedialibrary_posts'))) {
      self::ensureIndex('realmedialibrary_posts', 'fid', 'fid');
```

Reply:
Oops 🙈 thanks!